css/styles.css

```diff
@@ -0,0 +1,288 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #f4f5f7;
+}
+
+header {
+  background: #f72210;
+}
+
+img {
+  width: 150px;
+}
+
+.cloud {
+  font-size: 16px;
+}
+
+.subscribe {
+  padding: 5px;
+  vertical-align: middle;
+}
+
+.nav-brand-big {
+  width: 350px;
+}
```

Comment:
Please use bootstrap classes when possible. In this case, you could use something like col-2 to achieve the same look you have.

Reply:
could not add as a class in bootstrap